src/main/groovy/org/savantbuild/plugin/dep/idea/IDEAPlugin.groovy

```diff
@@ -139,12 +139,21 @@ class IDEAPlugin extends BaseGroovyPlugin {
 
   private toRelativePATH(Path path, String userHome) {
     def artifactRealPath = path.toRealPath().toString()
+    // If we have a cache process defined, getting the first one is useful because it gives a more precise
+    // location of where the .savant/cache directory is. In our tests, it's in ../../../savant-dependency-management
+    def cacheProcess = project.workflow.fetchWorkflow.processes.find { p -> p instanceof CacheProcess
+    } as CacheProcess
+    def cacheDir = cacheProcess ? new File(cacheProcess.dir) : null
     def projectRealPath = project.directory.toRealPath().toString()
 
     // Only perform a replace if the project path or user home are at the front of the real path
     // - While unlikely, a path could repeat, and we only want to replace the prefix of the path
 
-    if (artifactRealPath.startsWith(projectRealPath)) {
+    if (cacheDir && artifactRealPath.startsWith(cacheDir.canonicalPath)) {
```

Comment:
Looks ok to me, but it may be best to have Brian take a look as well. But it seems like the right thing to do.

Reply:
This looks pretty odd to me. The plugins shouldn't need to know where the fetch and publish processes are putting things. They are all relative to the file and module/project location. If this is just for testing, I'd revert it and have the tests use the relative paths, even if they don't look the best.

Reply:
The problem I am solving, with the impl change, was that previously, these tests would fail if you didn't clone the Savant code in exactly the same spot (since our test deps are in another repo (savant-dep-mgmt), the expected IMLs are user home dependent).

I changed the impl to find the cache dir, even if it was outside the project dir, which was 1 way of dealing with that. I just pushed (perhaps better way), which instead of modifying the impl, in the test, I just copy the deps from the savant-dep-mgmt repo to our project dir (`build/test`), which then ensures we don't fallback to `$USER_HOME$` with our test expecations.